packages/opentelemetry-metrics/src/Metric.ts

```diff
@@ -33,16 +33,16 @@ export abstract class Metric<T> implements types.Metric<T> {
   }
 
   /**
-   * Returns a Handle associated with specified label values.
+   * Returns a Handle associated with specified LabelSet.
    * It is recommended to keep a reference to the Handle instead of always
    * calling this method for each operation.
-   * @param labelValues the list of label values.
+   * @param labels the LabelSet used to associate with this metric handle.
    */
-  getHandle(labelValues: string[]): T {
-    const hash = hashLabelValues(labelValues);
+  getHandle(labels: types.LabelSet): T {
+    const hash = hashLabelValues(Object.values(labels));
```

Comment:
1. Take a look at what JS uses to determine uniqueness of an object inside a set/dictionary. Instead of calling a hash function like `hashLabelValues`, you can override that uniqueness function inside the `LabelSet` class and store the labels themselves instead of needing a hash.

2. If you do choose to stick with `hashLabelValues`, I would not use an comma delimiter to separate the values, as many common label values have commas in them (such as date).